packages/fortinet_fortigate/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.26.1"
```

Since a new field is being added, let's make this a minor change.

```suggestion
- version: "1.27.0"
```